utils/frame/remote-externalities/src/lib.rs

```diff
@@ -609,7 +608,14 @@ where
 
 		let mut sp = Spinner::with_timer(Spinners::Dots, "Inserting keys into DB...".into());
 		let start = Instant::now();
-		pending_ext.batch_insert(key_values.clone().into_iter().map(|(k, v)| (k.0, v.0)));
+		pending_ext.batch_insert(key_values.clone().into_iter().filter_map(|(k, v)| {
+			// Don't insert the child keys here, they need to be inserted seperately with all their
+			// data in the load_child_remote function.
+			match is_default_child_storage_key(&k.0) {
+				true => None,
+				false => Some((k.0, v.0)),
+			}
```

Comment:
```suggestion
			is_default_child_storage_key(&k.0).then_some((k.0, v.0))
```

Reply:
`then_some` returns the `Some` variant when the boolean is true, in this case we need to return it when it's false, so we should negate it first

Reply:
I played with this for a while and found I could simplify it down to 

```rust
		pending_ext.batch_insert(key_values.clone().into_iter().filter_map(|(k, v)| {
			// Don't insert the child keys here, they need to be inserted seperately with all their
			// data in the load_child_remote function.
			(!is_default_child_storage_key(&k.0)).then(|| (k.0, v.0))
		}));
```
which while more concise, I feel it is more difficult to read and imo not worth the few lines saved.